### ISSUE TITLE
Dedicated palette for streaming in players view (client.swf)

### ISSUE THREAD

**Issue by foobarlab** [foobarlab/UpStage-Video-Hack#38]
It would be nice to have a dedicated palette in the players view to have the possibility to change streaming settings on the fly (e.g. play or stop stream, seek to position, opacity, stream dimensions, etc...).

**Comment by helenvarley** [comment]
yes this would be really great; also resizing (altho i know that UpStage doesn't handle that - but let's keep it in mind as a future wish)

**Comment by foobarlab** [comment]
ok, let's see what we can do. resizing is not yet a general feature so it has further implications.
setting the given size of a video is implemented right now, so it resizes automatically to the origin  dimensions (e.g. if the video size is 320x240 or 640x480, or any other).

another setting to be able to change concerns audio, so it should be possible to mute sound and set the volume level.

in general it also sounds useful to allow looping of streams.
